compiler-core/src/docs.rs

```diff
@@ -222,30 +222,85 @@ pub fn generate_html(
         });
     }
 
-    // Render static assets
+    // Render static assets - TODO: do not push these through the templater
```

Comment:
What does this mean?

Reply:
This PR is not yet done entirely.
Currently all `OutputFile` are being pushed through the template parser as part of `generate_html` but:

- we do not need to push static assets through that
- we cannot push static binary assets like woff2 web fonts through that

Also notice: The google web font breaks privacy requirements in the EU

Reply:
Here `generate_html` is called:

```rust
pub(crate) fn build_documentation(
    config: &PackageConfig,
    compiled: &mut Package,
) -> Result<Vec<gleam_core::io::OutputFile>, Error> {
    compiled.attach_doc_and_module_comments();
    cli::print_generating_documentation();
    let mut pages = vec![DocsPage {
        title: "README".to_string(),
        path: "index.html".to_string(),
        source: paths::readme(), // TODO: support non markdown READMEs. Or a default if there is none.
    }];
    pages.extend(config.documentation.pages.iter().cloned());
    let outputs = gleam_core::docs::generate_html(config, compiled.modules.as_slice(), &pages);
    Ok(outputs)
}
```

Reply:
I will break this PR apart again, and remove the unused files and do the google web fonts thing as a separate PR